src/meta/src/hummock/manager/mod.rs

```diff
@@ -1150,12 +1150,8 @@ where
             }
             let is_success = if let TaskStatus::Success = compact_task.task_status() {
                 // if member_table_ids changes, the data of sstable may stale.
-                let is_expired = current_version
-                    .levels
-                    .get(&compact_task.compaction_group_id)
```

Comment:
Do we need to check CG existence here?

Reply:
I can add a `assert` here because we would remove group while this group is empty it means that all state-tables have been removed and we can found this task expired with `is_compact_task_expired `.

Reply:
And we would clear compact status when we found this group does not exist in `HummockVersion`, so a compact task for a removed group would be set to `InvalidGroupCanceled`.

Reply:
> I can add a `assert` here because we would remove group while this group is empty it means that all state-tables have been removed and we can found this task expired with `is_compact_task_expired `.

nice!